recipes/apoc/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "apoc" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/apoc-{{ version }}.tar.gz
+  sha256: 719c0a18e68d10392f46544028842249d109b0a7188dc8acb98c7d1615d382d4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.21
+    - pyclesperanto-prototype >=0.8.0
+    - python >=3.6
+    - scikit-learn >=0.24.2
+
+test:
+  imports:
+    - apoc
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - oclgrind  # [win]
```

Comment:
Can't we just use this one for all platforms?

Reply:
Yes we can!